src/LinkDotNet.NCronJob/NCronJobExtensions.cs

```diff
@@ -32,6 +32,9 @@ public static class NCronJobExtensions
         var builder = new NCronJobOptionBuilder(services, settings);
         options?.Invoke(builder);
 
+        // temporary, need to be replaced with a proper configuration
+        services.AddSingleton<ConcurrencyConfig>((sp) => new ConcurrencyConfig(3));
```

Comment:
By the way is this still valid for this PR?

Reply:
No, that must've made it in when I did the first merge. I'll remove it.

Reply:
this was removed